### PR TITLE
Add readme.md

### PR DESCRIPTION
**Pull request by siebenbrunner**

### CODE REVIEW

**Comment by rht** [comment]
Usually, a readme file is named `README.md` instead of `readme.md`.

File: abcFinance/abcfinanceagent.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+class AbcFinanceAgent(abcEconomics.Agent):
+    def __init__(self, *param, **kwparam):
+        super().__init__(*param, **kwparam)
+
+        self.accounts = Ledger(
+            residual_account_name=kwparam.get('residual_account_name', 'equity'))
+
+    def __getattr__(self, attribute):
+        if attribute in ['book_end_of_period', 'book', 'make_stock_accounts', 'make_flow_accounts',
```
Comment (rht) [inline abcFinance/abcfinanceagent.py]: What is the reasoning in limiting the attributes? I think it is common for any agent that inherits from an `AbcFinanceAgent` to have additional attributes.

**Comment by rht** [comment]
Should the `requirements.txt` contain `abcEconomics`?

File: abcFinance/abcfinanceagent.py (hunk 1 above)
Comment (siebenbrunner) [reply abcFinance/abcfinanceagent.py]: hmm, no idea, this was old code... I'm happy to change the Agent (Accountant?) class

**Comment by siebenbrunner** [comment]
> 
> 
> Should the `requirements.txt` contain `abcEconomics`?

You think it shouldn't? I'm happy to have a manually curated `requirements.txt`, I think they're cleaner than the ones using `pip freeze`.

**Pull request merged**